### ISSUE TITLE
Blokada bug with Facebook Messenger audio calling on cellular data

### ISSUE THREAD

**Issue by jolo283** [blokadaorg/blokada#694]
I was told to open an issue report here after communicating with the help desk (hello@blokada.org).

### Actual behaviour

I noticed that when using Blokada, my Facebook Messenger audio calls disconnect and reconnect every few seconds when on cellular data, making it impossible to use. I noticed that the LTE indicator in my status bar would disappear and reappear whenever I cut out as well. This issue doesn't occur with other apps while on cellular data.

Attempts at debugging:

- Bypassed Messenger app
- Force stopped and restarted Blokada

I can confirm that uninstalling/disabling Blokada removes this issue, so I'm certain it is the cause of this. I've uploaded a video demonstrating this behavior here: https://drive.google.com/file/d/1RuPIHlQ06XVahqADfoAJER2qb67HiuEv/view?usp=sharing

### Expected behaviour

Messenger audio calls should function normally when the app is whitelisted.

### Steps to reproduce  

1. Install Blokada
2. Whitelist Messenger
3. Place an audio call

### Configuration
1. Blokada version: 4.5.2
2. Device model: Galaxy S10e
3. Android version / ROM: Android 10, Kernel Version 4.14.117

### Questions
1. Do you know about recent updates (for example system update, security patch update)? 

Yes, my phone is fully up to date.

2.  How often does this problem happen to you?

Every time I try to place a Messenger audio call on cellular data.

**Comment by s4pun1s7** [comment]
Hello, and thank you for your report!

Would you mind sharing your log? I wasn't able to reproduce this issue on my android.

**Comment by jolo283** [comment]
Can I email the log? I'd rather not post it publicly.

**Comment by s4pun1s7** [comment]
Sure, feel free to mail me, I'll share with the relevant team martin.k4lchev@gmail.com

**Comment by jolo283** [comment]
Hi, I just emailed you the log!

**Comment by jolo283** [comment]
An additional note: I placed a Messenger audio call on data immediately before opening the log, so hopefully that should be reflected in the log as well.

**Comment by s4pun1s7** [comment]
Have you tried leaving Blokada on **BUT** w/o whitelisting Facebook Messenger (and all other related settings)?

**Comment by alexzhou12345** [comment]
Yes, leaving Blokada on and not whitelisting Messenger still results in this issue.

**Comment by jolo283** [comment]
Yes, leaving Blokada on and NOT bypassing Messenger still gives this issue.

**Comment by jolo283** [comment]
Hi @s4pun1s7 , just wanted to see if there were any updates!

**Comment by jolo283** [comment]
Hi @s4pun1s7  @peterroth , I wanted to follow-up on this, as the issue still persists even with the latest update. It seems like someone else has also encountered this issue as well (https://forum.xda-developers.com/pixel-4-xl/help/blokada-breaks-fb-messenger-calls-t4066613) so I don't think it's limited to just me.
